src/main/scala/org/scalameter/BasePerformanceTest.scala

```diff
@@ -2,13 +2,17 @@ package org.scalameter
 
 
 import java.util.Date
+import org.scalameter.picklers.Pickler
 import org.scalameter.utils.Tree
+import scala.language.existentials
 import scala.util.DynamicVariable
 
 
 
 abstract class BasePerformanceTest {
 
+  type Result
```

Comment:
I've used type alias, since that allows us to specify easily default measurer, aggregator, and so on in the `BasePerformanceTest` descendants.
One caveat: I think that Java users should have different API, but I don't know currently how to do it other way.